pkgs/build-support/emacs/wrapper.sh

```diff
@@ -23,6 +23,11 @@ else
     newLoadPath+=("")
 fi
 
+# NOTE: Even though we treat EMACSNATIVELOADPATH like EMACSLOADPATH in
```

Comment:
Can you squash this modification along with the other one https://github.com/NixOS/nixpkgs/pull/251285/commits/a3a144232d78831dbb170cf999206d7f071961f2 modifying the code?

Reply:
This comment has nothing to do with that bug fix.  They serve different purpose.  I do not think merging them is a good idea.